colors/Photon.gpl

```diff
@@ -0,0 +1,56 @@
+GIMP Palette
+Name: Firefox/Photon
+
+255 26 217 #ff1ad9
```

Comment:
We should name these as per the names in the json file (i.e. `blue-50`).

Reply:
I agree.

Reply:
I agree!